README.md

```diff
@@ -22,57 +30,92 @@
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-# 📖 [your_project_name] <a name="about-project">Hello Microverse</a>
 
-**[your_project__name]** This is a basic HTML & CSS project to practice all the tools that a wiil need to use in the future.
+# 📖 [Hello Microverse project] <a name="about-project"></a>
+
+
+**[Hello Microverse project]** is a basic Html and Css project made to practice with all the tools we will need ahead in the program. 
 
 ## 🛠 Built With <a name="built-with"></a>
 
+HTML5, CSS, Git and Github
 ### Tech Stack <a name="tech-stack"></a>
 
+
 <details>
-  <summary>Html</summary>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://lenguajehtml.com/html/">HTML5</a></li>
+    <li><a href="https://lenguajecss.com/">CSS3</a></li>
+  </ul>
 </details>
+
 <details>
-  <summary>Css</summary>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
 </details>
 
+
 ### Key Features <a name="key-features"></a>
 
-- **[Show the Hello Microverse message on the Web]**
-- **[Change the message color using Css]**
 
+- **[Best organization]**
+- **[Professional documentation]**
+- **[Clean Working]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+
 ## 💻 Getting Started <a name="getting-started"></a>
 
+
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
 
 In order to run this project you need:
 
-- Install Visual Studio Code 
+- Visual Studio Code installed
+- Npm installed
+- A Github account
+- Git installed
 
 ### Setup
 
 Clone this repository to your desired folder:
 
-  cd hello-world
-  git clone git@github.com:lRebornsl/hello-world.git
+  cd my-folder
+  git clone git@github.com:bohaz/Hello-.git
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] Kindly complete the Getting started section in your readme, you can follow the readme in [this repo](https://github.com/AdnanOlarmmi/Technopreneur) for inspiration.